pkg/server/plugin/upstreamauthority/disk/disk_test.go

```diff
@@ -250,7 +258,11 @@ func (s *DiskSuite) testCSRTTL(preferredTTL int32, expectedTTL time.Duration) {
 	csr, _, err := util.NewCSRTemplate(validSpiffeID)
 	s.Require().NoError(err)
 
-	resp, err := s.p.MintX509CA(ctx, &upstreamauthority.MintX509CARequest{Csr: csr, PreferredTtl: preferredTTL})
+	stream, err := s.p.MintX509CA(ctx, &upstreamauthority.MintX509CARequest{Csr: csr, PreferredTtl: preferredTTL})
+	s.Require().NoError(err)
+	s.Require().NotNil(stream)
+
+	resp, err := stream.Recv()
```

Comment:
We should add an additional call to Recv() somewhere after here that we assert returns io.EOF.